modules/op-resolver-cluster-tasks-ref.adoc

```diff
@@ -8,7 +8,7 @@
 
 An {pipelines-shortname} installation includes a set of standard tasks that you can use in your pipelines. These tasks are located in the {pipelines-shortname} installation namespace, which is normally the `openshift-pipelines` namespace. You can use the cluster resolver to access the tasks.
 
-`ClusterTask` functionality is deprecated since {pipelines-shortname} 1.10 and is planned for removal in a future release. If your pipelines use `ClusterTasks`, you can re-create them with the tasks that are available from the {pipelines-shortname} installation namespace by using the cluster resolver. However, certain changes are made in these tasks compared to the existing `ClusterTasks`.
+Until version 1.16, {pipelines-shortname} included `ClusterTask` functionality. Versions 1.17 and later no longer include this functionality. If your pipelines use `ClusterTask` references, you can re-create them with the tasks that are available from the {pipelines-shortname} installation namespace by using the cluster resolver. However, certain changes are made in these tasks compared to the previously existing `ClusterTask` definitions.
```

Comment:
```suggestion
Until version 1.16, {pipelines-shortname} included `ClusterTask` functionality. Versions 1.17 and later do not include this functionality. If your pipelines use `ClusterTask` references, you can re-create them with the tasks that are available from the {pipelines-shortname} installation namespace by using the cluster resolver. However, certain changes are made in these tasks compared to the previously existing `ClusterTask` definitions.
```
I would simplify this at some point for minimalism. It _sounds like_ 1.17 and later used to also have this function, but it was removed ("no longer"). If they never had it, I would just say they don't. Not required for merge, just a thought.